### ISSUE TITLE
Imperfect Regular Expression yields completely incorrect results

### ISSUE THREAD

**Issue by cjcarrick** [cjcarrick/YTClipsBySubs#1]
https://github.com/cjcarrick/YTClipsBySubs/blob/8f1e2436e621f81549c9d2b3a8e9b4b604e7d530/findSpeech.sh#L33-L37

So here's the main problem area: the regex. :/
I know it looks big, but it has to be. There's quite a bit of variation in what I need to scrape from these subtitle files, and you need to be specific and thorough about it. That's why there are so many or (`|`) statements. The upper regex is what I know works (test it for yourself using [the sample subtitle file](https://github.com/cjcarrick/YTClipsBySubs/blob/9fed60eb0aa8e4adcbb132a82695e6ff1053dc30/ssZ_8cqfBlE.en.vtt) on regex101 like I did). The output from `pcregrep` isn't quite right. From [the sample output](https://github.com/cjcarrick/YTClipsBySubs/blob/dc0dbf2db9077034259896b9da8cfa2b693a6995/ssZ_8cqfBlE-combined.txt):
https://github.com/cjcarrick/YTClipsBySubs/blob/dc0dbf2db9077034259896b9da8cfa2b693a6995/ssZ_8cqfBlE-combined.txt#L234-L245
The second regex pattern I came up with (line 37 of the script) was supposed to fix something, and I forget if that something was this, but obviously it wasn't enough. You can see after line 239 there is a missing bit of text, which the third part of the regex is supposed to match, but for whatever reason it doesn't. It may have something to do with the newline character, but I really don't know.

Oh also, when the output from `pcregrep` messes up just one time like it does here, it throws the whole rest of the script off, so this is pretty important.

Tl;dr: if there is a way to get exactly the regex that regex101 uses into bash somehow, that would be perfect. But I probably just need to fix the pattern I already came up with.
